<launch>

    <!-- LiDAR distance test -->
	<arg name="angle_ub" default="-150"/>
	<arg name="angle_lb" default="-160"/>

    <!-- <include file="$(find ydlidar_ros_driver)/launch/G6_G7.launch" /> -->

    <node pkg="sensor_analyst" type="lidar_filter_node" name="lidar_dist_filter">
        <param name="angle_ub" value="$(arg angle_ub)"/>
        <param name="angle_lb" value="$(arg angle_lb)"/>
        <!-- <remap from="scan" to="scan"/> -->
    </node>

    <node pkg="sensor_analyst" type="chassis_control_node" name="chassis_control_node">
        <param name="mode" value="1"/>
        <param name="width" value="2.0"/>
        <param name="speed_max" value="0.5"/>
    </node>

</launch>
